src/main/scala/org/broadinstitute/dsde/workbench/sam/schema/JndiSchemaDAO.scala

```diff
@@ -217,37 +204,33 @@ class JndiSchemaDAO(protected val directoryConfig: DirectoryConfig)(implicit exe
     }
   }
 
-  // Workbench Person
-
-  def createWorkbenchPersonSchema(): Future[Unit] = withContext { ctx =>
+  def createWorkbenchPetServiceAccountSchema(): Future[Unit] = withContext { ctx =>
     val schema = ctx.getSchema("")
 
-    createAttributeDefinition(schema, "1.3.6.1.4.1.18060.0.4.3.2.400", Attr.petServiceAccount, "pet service account of the user", true)
+    createAttributeDefinition(schema, "1.3.6.1.4.1.18060.0.4.3.2.70", Attr.project, "google project", true, equality = Option("caseIgnoreMatch"))
 
     val attrs = new BasicAttributes(true) // Ignore case
-    attrs.put("NUMERICOID", "1.3.6.1.4.1.18060.0.4.3.2.300")
-    attrs.put("NAME", "workbenchPerson")
+    attrs.put("NUMERICOID", "1.3.6.1.4.1.18060.0.4.3.2.700")
+    attrs.put("NAME", "petServiceAccount")
```

Comment:
Should there be an `Attr.petServiceAccount`? I think I saw it hardcoded in a few places.